src/lib/components/EventMap/index.js

```diff
@@ -66,7 +68,7 @@ class EventMap extends BaseStateElement {
 
   render() {
     if (!this.events) {
-      return '';
+      return html``;
```

Comment:
So returning just a string throws a hissy of typing errors. I could return `null` or `undefined`, would that solve the "falsey" issue @samthor.

Reply:
No, please return `html''`, but add a comment.

Returning null/falsey/empty string _tells Lit_ to leave the current contents alone. I've only seen it inside our codebase as a bug. This may very well be one of those cases.